workflows/biosiftr.nf

```diff
@@ -190,46 +192,62 @@ workflow SHALLOWMAPPING {
     INTEGRA_MODU(SM_COMM_KC.out.kegg_comp.collect { it[1] }, 'sm_modules')
     ch_versions = ch_versions.mix(INTEGRA_MODU.out.versions.first())
 
-    ch_dram_community = SM_FUNC.out.dram_comm.collectFile(name: 'dram_community.tsv', newLine: true) { it[1] }.map { dram_summary -> [[id: 'integrated'], dram_summary] }
+    if (params.run_dram) {
+        ch_dram_community = SM_FUNC.out.dram_comm.collectFile(name: 'dram_community.tsv', newLine: true) { it[1] }.map { dram_summary -> [[id: 'integrated'], dram_summary] }
+        DRAM_DISTILL(ch_dram_community, 'sm', 'community')
+        ch_versions = ch_versions.mix(DRAM_DISTILL.out.versions.first())
+    }
 
-    DRAM_DISTILL(ch_dram_community, 'sm', 'community')
+    // ---- MAPPING READS with bwamem2 (optional): mapping, cleaning output, and profiling ---- //
+    // bwamem2 optimisation depends on the number of species detected by sourmash
 
-    ch_versions = ch_versions.mix(DRAM_DISTILL.out.versions.first())
+    /************************************************************************************/
+    /* bwamem2 optimisation depends on the number of species detected by sourmash       */
+    /* If species number is > 150, then coverage threshold = 0.01                       */
+    /* If species number is <= 150, then coverage threshold = 0.1                       */
```

Comment:
I feel this should be part of the documentation

Reply:
That's in the new diagram and will be explained in the paper